### ISSUE TITLE
About evaluation results in the experiment

### ISSUE THREAD

**Issue by MingCongSu** [li-ronghui/LODGE#16]
Hello author, thank you for bringing us this fantastic work and implementation code 👍👍

I just have some questions about the evaluation and the result in the paper. 
In the paper "4.1. Experimental Settings - Implementation details" you mentioned that:
>  Finally, we interpolated the output dances to 60 fps and followed the experimental setup of Bailando [40] for testing.

In my understanding, Bailando used kinetic and geometric features from both generated dance and all of the motion sequences (including train and test data) to compute FID.

1. Did you follow the same setting here to produce the evaluation results on both AIST++ and FineDance datasets?
2. Can you provide more details about how to evaluate the model and reproduce the results in the paper?

Thanks a lot!

**Comment by li-ronghui** [comment]
Hello, MingCong

For the comparison on the AIST++ dataset, we used the same evaluation setting as Bailando.
For the comparison on the FineDance dataset, our evaluation setting can be found in the ./metric folder.
1. The metric calculation functions for both are essentially the same. However, due to differences in the datasets, there might be some minor variations in the experimental setup.
2. The training, generation, and metric testing on the FineDance dataset are all available already on this GitHub.  The samples in the AIST++ dataset are too short, making it unsuitable for long sequence dance generation tasks using Lodge. In our experiments on AIST++, we had to reduce the parameters for global and local windows. Additionally, many sample data were unusable in training due to their short duration.
